### DISCUSSION TITLE
This feed is full of unnecessary information, spammy & distracts from work - remove it

### DISCUSSION THREAD

**Discussion by hairyhoncho** [community/community#59486] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This feed is full of unnecessary information, spammy & distracts from work - remove it

**Comment by orubel** [comment]
do not want!!!

**Comment by David-at-TechCXO** [comment]
I agree with @hairyhoncho.

**Comment by cconvey** [comment]
Please stop trying to make Github into a social media platform.  Software development is hard enough without adding that dimension to our work.

**Comment by mhutfield-Itrinegy** [comment]
Agreed! Hate it.
